doc/man1/openssl.pod

```diff
@@ -106,7 +107,8 @@ CRL to PKCS#7 Conversion.
 
 =item B<dgst>
 
-Message Digest Calculation.
+Message Digest Calculation. MAC calculations are Superseded by
```

Comment:
Nitpicking: it seems the first letters of `Calculation` and `Superseded` should be lowercase, the same for the mac item below. But it's strange that many other items in this doc section have upper-cased the first letters of the words either...